src/compiler/cross-team-collaboration.md

```diff
@@ -0,0 +1,68 @@
+relevant parts to the compiler team, etc) and add the `I-compiler-nominated`
+label to a issue (you can use `@rustbot label +I-compiler-nominated` to do
+this).
+
+Once nominated, the issue will be discussed in a upcoming [triage
+meeting](./triage-meeting.html). The compiler team doesn't always get through
+all nominated issues each week, so it can take more than one meeting for your
+issue to be discussed.
+
+Once discussed, a member of the team will comment on the issue with the
+conclusion of the discussion and linking to the relevant Zulip chat.
+
+## ..to be fixed
+If there is an existing working relationship between a member of the requesting
+team and a contributor to the compiler, then the first option that a team has
+for requesting tasks be completed is to ping that contributor and ask if they
+can complete the task. It is recommended that pings take place in public Zulip
+channels so that..
+
+- ..other contributors that have free time have the opportunity to
+  offer their help.
+- ..other compiler team members/leadership can ensure that requests being made
+  are reasonable (see the rest of this section for the types of issues that the
+  compiler team commits to prioritizing on behalf of other teams).
+
+It is worth considering the available bandwidth of the contributor that the
+request is being made of, and whether their areas of expertise in the compiler
+are relevant.
+
+When there is not a appropriate contact in the compiler team to reach out to
+directly, write a comment on a GitHub issue (or create an issue) describing the
+task that needs completed. Teams should nominate issues for the compiler team
+when issues..
+
+- ..are not already tracked by/part of an existing initiative or working group
+  and..
+- ..are blocking/impeding the work of the other team (e.g. a feature or bug
+  preventing the stabilization of something otherwise complete), but..
+- ..aren't absolutely mission-critical - a soundness bug or otherwise critical
+  issue will be prioritized by the [prioritization working
+  group](./prioritization.html) and addressed through the compiler team's other
+  processes for these bugs.
+
+A detailed description of the feature being requested or the bug to be fixed is
+helpful wherever possible (so that the compiler contributor does not need to
+make a guess as to a solution that would solve the problem for the requesting
+team). If a member of the requesting team isn't explicitly listed as the
+point-of-contact for the issue, then the author of the comment will be assumed
+to be the point-of-contact.
+
+Add the `I-compiler-nominated` label to a issue (you can use `@rustbot label
++I-compiler-nominated` to do this).
+
+Once nominated, the issue will be assigned in a upcoming [triage
+meeting](./triage-meeting.html). The compiler team doesn't always get through
+all nominated issues each week, so it can take more than one meeting for your
+issue to be discussed.
```

Comment:
In general, this is what would usually happen with some kind of important request from another team but there is also the possibility that we need to reject a request (perhaps we have insufficient bandwidth vs critical/high priority compiler bugs, perhaps we aren't able to find anyone motivated to work on the specific issue at this time, perhaps there's something in the request which isn't feasible to implement at this time). 

I think there should be some mention of an alternative outcome where the issue is not assigned and instead, we pass the issue back to the other team with some kind of explanation of why we weren't able to assign the issue.

Reply:
Added some wording for this.